### PR TITLE
Implemented Student Id detection

### PR DESCRIPTION
**Pull request by Ltchance**

### CODE REVIEW

File: pii_scan.py [hunk 1]
```diff
@@ -54,6 +54,15 @@ def analyze_text(text: str, show_supported=False, show_details=False, score_thre
                                         patterns=[uuid_pattern])
     registry.add_recognizer(uuid_recognizer)
 
+
+    #Create an additional pattern to detect a 123456789 Student Id
+    student_id_pattern = Pattern(name='student_id',
+                           regex=r'\b[0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F]\b',
```
Comment (claesmk) [inline pii_scan.py:60]: In your issue description you say that a student ID is a 9 digit number

[0-9a-fA-F] will detect digits and letters, so, A23456789 would be detected as a STUDENT_ID which I don't think you want.

You probably want to do something like this which would detect only a digit 9 times
```
    student_id_pattern = Pattern(name='student_id',
                           regex=r'\b\d{9}\b',
                           score=0.8)
```

**Comment by Ltchance** [comment]
> Looks great except for my comment about digits and letters being detected.

I am having an issue syncing my new pull request. I think I am having a merge conflict that it isn't telling me about. Do you know what I can do?

**Comment by claesmk** [comment]
@Ltchance which new pull request are you referring to?  All you need to do is push the changes to your current branch and this pull request will update automatically

**Comment by Ltchance** [comment]
> @Ltchance which new pull request are you referring to? All you need to do is push the changes to your current branch and this pull request will update automatically

@claesmk I thought I did that but something went wrong. The only thing it will let me do is Sync it but that fails

**Review by claesmk — APPROVED** [review]
Looks good now - your branch was already synced correctly.  Not sure why you weren't able to push the changes we discussed.  I was able to do that OK.  Get one more person from your team to review so you can merge this PR

**Review by VincentGoins — APPROVED** [review]

**Pull request merged**